cli/lsp/language_server.rs

```diff
@@ -3503,25 +3527,30 @@ impl Inner {
     config_changed: bool,
   ) {
     self.project_version += 1; // increment before getting the snapshot
-    self.ts_server.project_changed(
-      self.snapshot(),
-      modified_scripts,
-      config_changed.then(|| {
-        self
-          .config
-          .tree
-          .data_by_scope()
-          .iter()
-          .map(|(s, d)| (s.clone(), d.ts_config.clone()))
-          .collect()
-      }),
-    );
+    if self.ts_server_initialized() {
+      self.ts_server().project_changed(
```

Comment:
Ah, need to coalesce change notifs outside of the server

Reply:
No... I should just call `ts_server.start()` lazily instead of having `OnceCell<TsServer>`.